docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
+| field           | `{"type":"field","fieldName":["score"]}`                          | `FieldReferenceDTO.of("score")`                                                                         | `score`                  | field reference value `score`  |
+| function        | `{"type":"function","functionName":"hour","fieldName":["score"]}` | `new FuncExpressionDTO.Builder()<br/>.withFunctionName("hour")<br/>.withFunctionArgs("score").build()`  | `hour(score)`            | function value `hour(score)`   |
+| constant        | `{"type":"constant","value":10, "dataType": "integer"}`           | `new LiteralDTO.Builder()<br/>.withValue("10")<br/>.withDataType(Types.IntegerType.get())<br/>.build()` | `10`                     | Integer constant `10`          |
+
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+{
+  "strategy": "hash",
+  "number": 4,
+  "funcArgs": [
+    {
+      "type": "field",
+      "fieldName": ["score"]
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+ new DistributionDTO.Builder()
+    .withStrategy(Strategy.HASH)
+    .withNumber(4)
+    .withArgs(FieldReferenceDTO.of("score"))
+    .build()
+```
+
+</TabItem>
+</Tabs>
+
+
+#### Sorted order table
+
+To define a sorted order table, you should use the following three components to construct a valid sorted order table.
+
+- Direction.  It defines in which direction we sort the table.
+
+| Direction  | Json   | Java                       | Description                               |
+|------------| ------ | -------------------------- |-------------------------------------------|
+| ascending  | `asc`  | `SortDirection.ASCENDING`  | Sorted by a field or a function ascending |
+| descending | `desc` | `SortDirection.DESCENDING` | Sorted by a field or a function ascending |
+
+- Null ordering. It describes how to handle null value when ordering
+
+| Null ordering Type | Json          | Java                       | Description                       |
+|--------------------| ------------- | -------------------------- |-----------------------------------|
+| null_first         | `nulls_first` | `NullOrdering.NULLS_FIRST` | Put null value in the first place |  
+| null_last          | `nulls_last`  | `NullOrdering.NULLS_LAST`  | Put null value in the last place  |
+
+- Sort term. It shows which field or function should be used to sort the table, please refer to the `Expression type` in the bucketed table chapter.
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+ {
+  "direction": "asc",
+  "nullOrder": "NULLS_LAST",
+  "sortTerm":  {
+    "type": "field",
+    "fieldName": ["score"]
+  }
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+ new SortOrderDTO.Builder()
+    .withDirection(SortDirection.ASCENDING)
+    .withNullOrder(NullOrdering.NULLS_LAST)
+    .withSortTerm(FieldReferenceDTO.of("score"))
+    .build()
+```
+
+</TabItem>
+</Tabs>
+
+
 :::tip
```

Comment:
Shouldn't we also introduce the default value situation?

Reply:
Have we supported `default` value completely?

Reply:
I mean the default sort order

Reply:
The default value of Partitioning, bucketing and sort order are `null` or `NONE`. Shall we need to emphasize the default null value?

Reply:
No, Partitioning, Bucketing, and Sortorder are all optional, users may choose not to concern themselves with these if they are not needed.

I have commented on the wrong line, there should be lines 844 to 851, `Direction` has default `Null ordering`

Reply:
OK

Reply:
I have added 
```text
Note: If the direction value is `ascending`, the default ordering value is `nulls_first` and if the direction value is `descending`, the default ordering value is `nulls_last`.
```